src/main/java/net/smacke/jaydio/OpenFlags.java

```diff
@@ -15,20 +15,38 @@
  */
 package net.smacke.jaydio;
 
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+
 /**
  * Constants for {@link DirectIoLib#oDirectOpen(String, boolean)}. </p>
  *
  * @author smacke
  *
  */
 public final class OpenFlags {
-    public static final int O_RDONLY = 00;
-    public static final int O_WRONLY = 01;
-    public static final int O_RDWR = 02;
-    public static final int O_CREAT = 0100;
-    public static final int O_TRUNC = 01000;
-    public static final int O_DIRECT = 040000;
-    public static final int O_SYNC = 04000000;
+    public static final int O_RDONLY = getOpenFlags("O_RDONLY");
```

Comment:
This will cause classloader failure if your new gcc-invoking command can't be executed. This change will make it only usable on unix systems with gcc and c stdlib installed. See request for changes below.